lms/urls.py

```diff
@@ -143,6 +143,9 @@
 
     url(r'^dashboard/', include('learner_dashboard.urls')),
     url(r'^api/experiments/', include('experiments.urls', namespace='api_experiments')),
+
+    # colaraz urls
```

Comment:
We usually use "[COLARAZ_CUSTOM]" keyword where ever we do some custom changes. With that, we write any description we want.